crates/frontend/src/components/function_form/function_form.rs

```diff
@@ -171,7 +171,7 @@ where
 pub fn test_form<NF>(
     function_name: String,
     stage: String,
-    handle_submit: NF,
+    _handle_submit: NF,
```

Comment:
If this is not being used why keep it. Or if you want to keep it make it optional.

Reply:
@pratikmishra356 , can you answer why you kept this handle_submit if its not being used?

Reply:
@pratikmishra356 , can you help here, why do we need this prop if this not being used anywhere

Reply:
@ShubhranshuSanjeev we dont need this , you can remove